src/main/java/ca/wolfram/beta/symath/operations/PowerNode.java

```diff
@@ -0,0 +1,51 @@
+package ca.wolfram.beta.symath.operations;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import ca.wolfram.beta.symath.MathNode;
+import ca.wolfram.beta.symath.NodeType;
+import ca.wolfram.beta.symath.VMap;
+
+public class PowerNode extends OperationNode{
+
+    PowerNode(List<MathNode> children) {
+        super(children);
+    }
+
+    /**
+     * Index 0 is the base and Index 1 is the power.
+     * @throws RuntimeException if number of nodes is incorrect.
+     */
+    public static MathNode create(List<MathNode> nodes) {
+        if (nodes.size() != 2) throw new RuntimeException("Power Node takes exactly 2 node");
+        return new PowerNode(nodes);
+    }
+
+    public static MathNode create(MathNode... nodes) {
+        return create(new ArrayList<>(Arrays.asList(nodes)));
+    }
+
+    @Override
+    public boolean simplify() {
+        //TODO
+        return super.simplify();
+    }
+
+    @Override
+    public NodeType getType() {
+        return NodeType.POWER;
+    }
+
+    @Override
+    public double operationEval(VMap map) {
+        return Math.pow(getChildren().get(0).eval(map), getChildren().get(1).eval(map));
+    }
+
+    @Override
+    public String toString() {
+        return String.format("(%s ^ %s)", getChildren().get(0).toString(), getChildren().get(1).toString());
```

Comment:
minor preference, but not sure if we want spaces or not. For example (x + 2) ^ (5x) vs (x + 2)^(5x)